capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,17 @@
+import java.util.List;
+
+interface CollegeScoreCardInterface {
+  /*General Use Functions*/
+  public boolean validateCollegeName(String collegeName);
+
+  public List<UniversityData> getCollegesData(List<String> collegeNames);
+
+  /*Search Feature Functions*/
```

Comment:
I'm not sure why the Java linter isn't working, but the formatting on these comments looks off (there should be spaces between the comment delimiters and the comment). Anyhow - these public methods should have [Javadoc](https://google.github.io/styleguide/javaguide.html#s7-javadoc). Can you add that?

Reply:
done! I think I did it right-let me know!